shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -338,6 +338,7 @@ - (void)pushRoute:(NSString*)route {
     messageLabel.textAlignment = NSTextAlignmentCenter;
     messageLabel.autoresizingMask =
         UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
+    messageLabel.textColor = UIColor.blackColor;
```

Comment:
nit/optional: tiny upgrade to consider: UIColor.label here and UIColor.systemBackground on line 329

Reply:
Right. You could also make the container have UIColor.systemBackgroundColor and the label take UIColor.labelColor to be fully dark-mode compatible. Don't forget to put the the setters behind an `@available(iOS 13.whatever)` if you do though.

Reply:
Updated to make the container background `systemBackgroundColor`.  I didn't bother making the label black on <iOS 13 since this label only shows up on iOS 14 and later.
Updated the description screenshot.
I don't think I need to specify the `labelColor`, that's the default, right?